cypress/integration/patch/task-table.ts

```diff
@@ -0,0 +1,205 @@
+/// <reference types="Cypress" />
+/// <reference path="../support/index.d.ts" />
+
+import {
+  clickOnPageBtnAndAssertURLandTableResults,
+  clickOnPageSizeBtnAndAssertURLandTableSize,
+} from "../../utils";
+
+const pathTasks = `/patch/5e4ff3abe3c3317e352062e4/tasks`;
+
+describe("Task table", () => {
+  before(() => {
+    cy.login();
+  });
+
+  beforeEach(() => {
+    cy.listenGQL();
+    cy.preserveCookies();
+  });
+
+  it("Updates the url when column headers are clicked", () => {
+    cy.visit(pathTasks);
+
+    cy.get("th.cy-task-table-col-NAME").click();
+    locationHasUpdatedParams("NAME", "ASC");
+
+    cy.get("th.cy-task-table-col-NAME").click();
+    locationHasUpdatedParams("NAME", "DESC");
+
+    cy.get("th.cy-task-table-col-NAME").click();
+    locationHasUpdatedParams("NAME", "DESC");
+
+    cy.get("th.cy-task-table-col-VARIANT").click();
+    locationHasUpdatedParams("VARIANT", "ASC");
+
+    cy.get("th.cy-task-table-col-VARIANT").click();
+    locationHasUpdatedParams("VARIANT", "DESC");
+
+    cy.get("th.cy-task-table-col-VARIANT").click();
+    locationHasUpdatedParams("VARIANT", "DESC");
+  });
+
+  it("Clicking task name goes to task page for that task", () => {
+    cy.visit(pathTasks);
+    cy.get("td.cy-task-table-col-NAME:first").within(() => {
+      cy.get("a")
+        .should("have.attr", "href")
+        .and("include", "/task");
+    });
+  });
+
+  it("Task count displays total tasks", () => {
+    cy.visit(pathTasks);
+    cy.waitForGQL("PatchTasks");
+    cy.get("[data-cy=total-task-count]").contains("47");
+  });
+
+  it("Sort buttons are disabled when fetching data", () => {
+    cy.visit(pathTasks);
+    cy.contains(TABLE_SORT_SELECTOR, "Name").click();
+    cy.once("fail", (err) => {
+      expect(err.message).to.include(
+        "'pointer-events: none' prevents user mouse interaction."
+      );
+    });
+  });
+
+  // flakey test - must fix
+  ["NAME", "STATUS", "BASE_STATUS", "VARIANT"].forEach((sortBy) => {
```

Comment:
You can try using a combination of the cypress `.each` command with a promise to iterate through this array and run each test with out them overlapping each other
https://docs.cypress.io/api/commands/each.html#Promises

Reply:
this test flaked because `clickSorterAndAssertTasksAreFetched` is unreliable. How can you tell if the tests are overlapping? From the test runner, I can see that the tests are running synchronously.